modules/ignition/assets.tf

```diff
@@ -64,6 +64,41 @@ data "ignition_file" "ntp_dropin" {
   }
```

Comment:
Please run `terraform fmt` on the root directory. There are a few minor formatting issues.

Reply:
Done